.rubocop.yml

```diff
@@ -1,9 +1,7 @@
 AllCops:
-  Include:
-    - '**/Rakefile'
-    - '*.gemspec'
   Exclude:
     - 'pkg/**/*'
+  TargetRubyVersion: 2.5
```

Comment:
I assume, you copied this from prawn-dev but we probably should update this. Our general policy is to only support live Rubies. 2.5 was EOLed last year so we need to update our CI matrices and such in all repos.

Speaking of prawn-dev. Maybe we should use it in this repo too?

Reply:
@pointlessone  So I think that's a good policy, but the code base doesn't really reflect that.  If that's the policy then I would recomend:

1. Update the required_gem_version in all of the gemspecs to reflect that (`>= 2.6.0`)
2. For all .rubocop.yml files, set TargetRubyVersion appropriately

Also note that Ruby 2.6 will be EOLed in about a month, so would that policy involve updating to a 2.7 minimum at that time?

Reply:
Well, no one had time to update all those files. That's what I usually do before an occasional release.

I believe 2 is handled in prawn-dev in the repos that have it as a dependency. prawn-dev is there to make it easier to have a common dev dependencies and configuration.

Yes, we'll drop 2.6 when it'll reach EOL. I wouldn't go straight to 2.7 right now. Maybe we'll have release before we can drop 2.6.

Reply:
That's fine.

Re: 1, if you want a more broad update to 2.6 and up, that's easy enough to do.  All the repos are fairly out of date in this regard - for example the gemspec entry here is `spec.required_ruby_version = '>= 1.9.3'`.  A sweep over the repo to update that to `spec.required_ruby_version = '>= 2.6.0'`

Re: 2, Ok.  It's easy enough to update this to depend on prawn-dev as a dependency.  Updating the prawn-dev TargetRubyVersion will likely require changes to all the downstream libraries, as the Rubocop behavior will change.

Reply:
1. Yes that would be great. This repo in particular doesn't get updates even as often as Prawn. That's why it's Ruby requirement is so out of whack.
2. That was the idea. We bump Rubocop/RSpec/whatever in prawn-dev. Cut a gem. Bump it in all the repos and fix whatever is broken. It is tedious but it's easier that way to keep configs/dev-deps consistent.

To be clear, it is not required for a release. I'd rather focus on bugfixes and feature PRs. But if you feel more comfortable with this I won't protest. 🙂